### ISSUE TITLE
Yellow Border not really visible on different devices

### ISSUE THREAD

**Issue by teguria** [IBVSoftware/CollaBoard-Feedback#227]
V. 77
The yellow border, although nice, it's hard to see in different devices and when you have a lot of light in the room. Plus, if you have color contrasts on some tiles, the yellow border is not visible.
I would recommend to go back to the red color (or another strong color) so that it'll be visible to everyone

**Comment by teguria** [comment]
works good now. closing
